### PR TITLE
Add anharmonic freq. and constrained scan calc. files to expand test …

### PR DESCRIPTION
**Pull request by Schamnad**
…suite

Gaussian 09 output and checkpoint files to expand the standard cclib test suite to include anharmonic frequency calculations and constrained scan calculations.

### CODE REVIEW

**Comment by langner** [comment]
Could you also add the input files?

**Comment by Schamnad** [comment]
@ghutchis could you provide the input files as well.

**Comment by langner** [comment]
Where do we stand with this?

**Comment by berquist** [comment]
It looks like tests need to be written, and we should probably have some basic ones go in with the files right away, rather than wait until later.

**Comment by langner** [comment]
Yes, I see. But we also need the input files.

**Comment by ghutchis** [comment]
These are the input files - sorry.

[ch3oh.com.gz](https://github.com/cclib/cclib/files/384293/ch3oh.com.gz)
[water-pes.com.gz](https://github.com/cclib/cclib/files/384294/water-pes.com.gz)

**Comment by langner** [comment]
Pushing back again.

**Comment by Schamnad** [comment]
Apologies, had lost track of this pull request. Committed the input files provided by Geoff

**Comment by langner** [comment]
So what is this testing beyond the current unit tests?

File: data/Gaussian/basicGaussian09/ch3oh-freq.com [hunk 1]
```diff
@@ -0,0 +1,13 @@
+%nproc=2
+#t B3LYP/6-31G* Opt
```
Comment (TDI-tenderholt) [inline data/Gaussian/basicGaussian09/ch3oh-freq.com]: Why ```Opt``` and not ```Freq```?

**Comment by Schamnad** [comment]
The two output files contain attributes which previously weren't present in any of the cclib data files.

1. vibanharms
2. scanparm

I am unsure of how to write the test cases for these attributes as the cclib documentation doesn't contain much information regarding these two attributes.

**Comment by langner** [comment]
I think we need to revisit whether we need those test files in the main repo or as regressions.

**Comment by langner** [comment]
Not sure what the status is here, so punting to v1.5.4.

**Comment by langner** [comment]
This change has become stale - moved the files into two new separate issues, #499 and #500.

**Pull request closed without merging**